styles/style.css

```diff
@@ -65,8 +93,41 @@ body {
 /* Jumbotron */
 .jumbotron {
 	border-radius: 0;
+	/*background-image: url(../images/header-bike.jpg);*/
+    background-size: cover;
+    background-repeat: no-repeat;
+	background-position: center bottom;
+	color: white;
+	background: linear-gradient( rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7) ), url('../images/header-bike.jpg');	
+	opacity: 0.8;
+}
+/* newly added boxes*/
+.my-row { 
+	border: 1px solid #939393;
+	background-color: #ededed;
+	height: 10rem;
+	font-weight: bold;
+	font-size: 1.5rem;
+	display: flex;
+	text-align: center;
+	align-items: center;
+	margin-bottom: 3rem;
+	
 }
 
+article.bg-grey {
+	background-color: lightgray;
+	padding: 2rem;
+
+}
+
+article.bg-light {
+	border: 1px solid #939393;
+	padding: 2rem;
+	margin-left: 2rem;
+}
```

Comment:
You should be adding horizontal padding to the same level as your `col-...` classes as it will break the grid. The trick to avoid the 2 columns touching, is to move all content, borders and background colours to one level down (or move the `col-` up one level). You'll need to add a wrapping `div` somewhere to do this...

Reply:
Okay..thanks for the feedback will look into this now